tests/sass/function-name-format.scss

```diff
@@ -0,0 +1,12 @@
+$foo: kabab-case();
+$foo: snake_case();
+$foo: camelCase();
+$foo: PascalCase();
+$foo: Camel_Snake_Case();
+$foo: SCREAMING_SNAKE_CASE();
+$foo: _with-leading-underscore();
+$foo: _does_NOT-fitSTANDARD();
+
```

Comment:
Would be worth adding a function declaration into these just to make sure we're catching those in the tests going forward.

```scss
@function cp($target, $container) {
  @return calc-percent($target, $container);
}
```